index.html

```diff
@@ -208,9 +208,46 @@ <h3 class="cardHeader aboutmeHeader">Skills</h3>
 
     </section>
 
-    
+    <section class="contactme centeredGrid">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form action="https://formspree.io/f/mleaonkj" method="POST" id="contactmeForm">
+            <input class="userinput" type="text" name="name" id="userName" maxlength="30" placeholder="Full name" required>
+            <input class="userinput" type="email" name="email" id="userEmail" placeholder="Email address" required>
+            <textarea class="userinput" name="message" id="userMessage" cols="30" rows="10" maxlength="500" placeholder="Enter your message" required></textarea>
+        </form>
+        <button class="button" type="submit" form="contactmeForm">Get In touch</button>
```

Comment:
- [ ] It would be better if we implement user interactions, for example; let u us enable `cursor: pointer` so that users can know when to click on buttons or when not to click